webrender/src/hit_test.rs

```diff
@@ -265,6 +267,11 @@ impl HitTester {
                     None => continue,
                 };
 
+                // Don't hit items with backface-visibility:hidden if they are facing the back.
+                if !item.is_backface_visible && transform.is_backface_visible() {
```

Comment:
We should cache the value of `transform.is_backface_visible()` above since it calls `Transform3D::determinant()` which is a fairly expensive operation.

Reply:
Fixed. I also took care not to compute `transform.is_backface_visible()` eagerly in the case where none of the items are backface-visibility:hidden (which is the common case).